doc/pod1/cellcc_config.pod

```diff
@@ -196,6 +196,12 @@ authenticate to AFS in order to run authenticated B<vos> commands.
 This defaults to B</etc/cellcc/vos.keytab>. This has no effect if localauth
 mode is turned on.
 
+=item B<vos/flags/release>
+
+This can be used to pass extra options to vos release. As an example, if the
+system administrator wants to enable dpf, we would have to set
+B<vos/flags/release/dpf>.
```

Comment:
Don't use dpf as an example; that's an experimental feature that only exists in site-specific deployments. We should also provide an actual example of what this looks like in the config file; it's not completely clear how to give options as key/val pairs (I know the existing documentation isn't good about providing concrete examples; we should start doing better). Suggested text:

This can be used to pass extra options to the "vos release" command line when CellCC releases a volume. The options must be given as an associative array as accepted by AFS::Command, with the option name as the key and its argument as the value. For example, to pass "-encrypt -config /usr/vice/etc", specify the following:

{
  vos: {
    flags: {
      release: {
        encrypt: True,
        config: "/usr/vice/etc",
      }
    }
  }
}

...or fix that with whatever syntax actually works in the config file :)

Reply:
Done.